website/src/_posts/2019-05-11-beam-summit-europe-2019.md

```diff
@@ -0,0 +1,88 @@
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Keep an eye out for a meetup in [Paris](https://www.meetup.com/Paris-Apache-Beam-Meetup). 
+
+If you are interested in starting your own meetup, feel free [to reach out](https://beam.apache.org/community/contact-us)! Good places to start include our Slack channel, the dev and user mailing lists, or the Apache Beam Twitter.
+
+Even if you can’t travel to these meetups, you can stay informed on the happenings of the community.  The talks and sessions from previous conferences and meetups are archived on the [Apache Beam YouTube channel](https://www.youtube.com/c/ApacheBeamYT). If you want your session added to the channel, don’t hesitate to get in touch! And in case you want to attend the next Beam event in style, you can also order your swag on the [Beam swag store](https://store-beam.myshopify.com)
+
+## Summits
+The first summit of the year will be held in Berlin:
+
+<img src="{{ "/images/blog/banner.jpg" | prepend: site.baseurl }}" alt="Beam Summit Europe Banner" >
+
+You can find more info on the [website](https://beamsummit.org) and read about the inaugural edition of the Beam Summit Europe [here](https://beam.apache.org/blog/2018/10/31/beam-summit-aftermath.html). At these summits, you have the opportunity to meet with other Apache Beam creators and users, get expert advice, learn from the speaker sessions, and participate in workshops. 
+
+We strongly encourage you to get involved again this year! You can participate in the following ways for the upcoming summit in Europe:
+
+🎫 If you want to secure your ticket to attend the Beam Summit Europe 2019, check our [event page](http://beam-summit-europe.eventbrite.com).
+
+💸 If you want to make the Summit even **more** awesome, check out our [sponsor booklet](https://drive.google.com/file/d/1R3vvOHihQbpuzF2aaSV8WYg9YHRmJwxS/view)!
+
+We also launched the CfP for our Beam Summit in North America, which will be held in collaboration with [ApacheCon](https://www.apachecon.com).
+
+🎤 If you want to give a talk, take a look at our [CfP](https://www.apachecon.com/acna19/cfp.html).
+
+Stay tuned for more information on the summit in North America and Asia.
+  
+## Why community engagement matters
+Why we need a strong Apache Beam community:
+* We’re gaining lots of code contributions and need committers to review them
```

Comment:
Maybe: "We’re receiving lots of code contributions and need committers to review and help new contributors overall."